python/meterpreter/ext_server_stdapi.py

```diff
@@ -2146,6 +2150,75 @@ def stdapi_net_config_get_routes(request, response):
         response += tlv_pack(TLV_TYPE_NETWORK_ROUTE, route_tlv)
     return ERROR_SUCCESS, response
 
+def windll_route_add_remove(is_add, request, response):
+    class IPAddr(ctypes.Structure):
+        _fields_ = [
+            ("S_addr", ctypes.c_ulong)]
+
+    def inet_addr(ip):
+        return IPAddr(struct.unpack("L", socket.inet_aton(ip))[0])
```

Comment:
This should use the same logic to convert an IPv4 address to an integer as L2185-2187.

Reply:
Actually they are same. You mean `"L"` formatter?

Reply:
Fixed in order to next suggestion ;)